code/game/area/Space_Station_13_areas.dm

```diff
@@ -529,6 +529,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/shuttle/ftl/cargo/mining
 	name = "Mining"
 	icon_state = "mining"
+	has_gravity = TRUE //The FOB shuttle has sci fi magic gravity

```

Comment:
why?

Reply:
Otherwise it doesn't have gravity in transit space. Unless it's remapped to have a grav gen on the Aetherwhisp and Scarab, this is it.

Reply:
Yes having no gravity while in transit is fine behavior for something with no gravity gen